ground/gcs/src/plugins/coreplugin/mainwindow.cpp

```diff
@@ -138,6 +130,31 @@ MainWindow::MainWindow() :
 #endif /* Q_OS_MAC */
     m_toggleFullScreenAction(0)
 {
+    // keep this in sync with main() in app/main.cpp
+    m_settings = new QSettings(XmlConfig::XmlSettingsFormat, QSettings::UserScope,
```

Comment:
so I understand, this file on disk is never truly used? it is just required in order to use the QSettings object? and if settings are changed on the fly, are they written to this file?not that it is bad to have the autosaved settings hanging around.